src/Microsoft.Identity.Client/Cache/TokenCacheDictionarySerializer.cs

```diff
@@ -49,23 +72,23 @@ public byte[] Serialize()
             }
 
             // reads the underlying in-memory dictionary and dumps out the content as a JSON
-            Dictionary<string, IEnumerable<string>> cacheDict = new Dictionary<string, IEnumerable<string>>
+            var cacheDict = new Dictionary<string, IEnumerable<string>>
             {
                 [AccessTokenKey] = accessTokensAsString,
                 [RefreshTokenKey] = refreshTokensAsString,
                 [IdTokenKey] = idTokensAsString,
                 [AccountKey] = accountsAsString
             };
 
-            return JsonHelper.SerializeToJson(cacheDict).ToByteArray();
+            return JsonHelper.SerializeToJson(cacheDict)
+                             .ToByteArray();
```

Comment:
>.ToByteArray(); [](start = 29, length = 15)

nit: It's a bit of a mystery what triggers a line shift. other places it seems like you are putting things back into one line. This one looks like it fits one line perfectly to me.

Reply:
it's resharper.  some files get formatted and some i don't touch so as to not have a bunch of whitespace.  i'm holding on formatting alignment again until vs2019 stabilizes a bit further and we can just standardize on .editorconfig files that all of us can use without resharper.